packages/react-table/src/components/TableComposable/examples/ComposableTableNestedTableExpandable.tsx

```diff
@@ -0,0 +1,189 @@
+          </Tr>
+        ))}
+      </Tbody>
+    </TableComposable>
+  );
+};
+
+export const ComposableTableExpandable: React.FunctionComponent = () => {
+  // In real usage, this data would come from some external source like an API via props.
+  const repositories: Repository[] = [
+    { name: 'Node 1', branches: '10', prs: '2', nestedComponent: <NestedReposTable />, link: <a>Link 1</a> },
+    { name: 'Node 2', branches: '3', prs: '4', link: <a>Link 2</a> },
+    {
+      name: 'Node 3',
+      branches: '11',
+      prs: '7',
+      nestedComponent: (
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+          consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+          est laborum.
+        </p>
+      ),
+      link: <a>Link 3</a>
+    },
+    {
+      name: 'Node 4',
+      branches: '11',
+      prs: '7',
+      nestedComponent: 'Expandable row content has no padding.',
+      link: <a>Link 4</a>,
+      noPadding: true
+    }
+  ];
+
+  const columnNames = {
+    name: 'Repositories',
+    branches: 'Branches',
+    prs: 'Pull requests',
+    link: 'Link',
+    action: 'Action'
+  };
+  // In this example, expanded rows are tracked by the repo names from each row. This could be any unique identifier.
+  // This is to prevent state from being based on row order index in case we later add sorting.
+  // Note that this behavior is very similar to selection state.
+  const initialExpandedRepoNames = repositories.filter(repo => !!repo.nestedComponent).map(repo => repo.name); // Default to all expanded
+  const [expandedRepoNames, setExpandedRepoNames] = React.useState<string[]>(initialExpandedRepoNames);
+  const setRepoExpanded = (repo: Repository, isExpanding = true) =>
+    setExpandedRepoNames(prevExpanded => {
+      const otherExpandedRepoNames = prevExpanded.filter(r => r !== repo.name);
+      return isExpanding ? [...otherExpandedRepoNames, repo.name] : otherExpandedRepoNames;
+    });
+  const isRepoExpanded = (repo: Repository) => expandedRepoNames.includes(repo.name);
+
+  const defaultActions = (repo: Repository): IAction[] => [
+    {
+      title: 'Some action',
+      onClick: () => console.log(`clicked on Some action, on row ${repo.name}`)
+    },
+    {
+      title: <a href="https://www.patternfly.org">Link action</a>
+    },
+    {
+      isSeparator: true
+    },
+    {
+      title: 'Third action',
+      onClick: () => console.log(`clicked on Third action, on row ${repo.name}`)
+    }
+  ];
+
+  return (
+    <TableComposable aria-label="Simple table">
+      <Thead>
+        <Tr>
+          <Th />
```

Comment:
Empty `th` cells should be `td`s

```suggestion
          <Td />
```

Reply:
Thanks! fixed